ksql-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -154,6 +151,22 @@ public void shouldFailCreateExistingTopic() {
     verify(adminClient);
   }
 
+  @Test
+  public void shouldFailCreateTopicNoAclsSet() {
+    expectedException.expect(TopicAuthorizationException.class);
```

Comment:
Our convention is to use a pattern of 
```
// Given:

// When:

// Then:
```
to structure our tests in order to make them more readable. Or in the case of tests that expect an exception, the pattern is
```
// Given:

// Expect:

// When:
```
See https://github.com/confluentinc/ksql/blob/6dd21fd7e7559e290a09ca59b7f0c45e5f328943/ksql-common/src/test/java/io/confluent/ksql/function/UdfIndexTest.java for examples of both.